st2common/st2common/triggers.py

```diff
@@ -32,75 +33,85 @@
 
 LOG = logging.getLogger(__name__)
 
-ACTION_SENSOR_ENABLED = cfg.CONF.action_sensor.enable
-TRIGGER_TYPE_ENDPOINT = cfg.CONF.action_sensor.triggers_base_url
-HTTP_POST_HEADER = {'content-type': 'application/json'}
-RETRY_WAIT = cfg.CONF.action_sensor.retry_wait
-TIMEOUT = cfg.CONF.action_sensor.request_timeout
-MAX_ATTEMPTS = cfg.CONF.action_sensor.max_attempts
 
+class InternalTriggerTypesRegistrar(object):
+
+    def __init__(self):
+        self._action_sensor_enabled = cfg.CONF.action_sensor.enable
+        self._trigger_type_endpoint = cfg.CONF.action_sensor.triggers_base_url
```

Comment:
Eventually we should probably switch to `auth.api_url` setting so we don't need to have 10 different settings which basically refer to the same thing.